src/Symfony/Component/Console/Attribute/Input.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Symfony\Component\Console\Attribute;
+
+use Symfony\Component\Console\Attribute\Reflection\ReflectionMember;
+use Symfony\Component\Console\Exception\LogicException;
+use Symfony\Component\Console\Input\InputInterface;
+
+#[\Attribute(\Attribute::TARGET_PARAMETER | \Attribute::TARGET_PROPERTY)]
+class Input
```

Comment:
Can we please make it final? This class is likely to qualify as a value resolver which I'm working on. The API is nice btw and we might end up with exactly this, but that would help with BC in case it slightly differs

Reply:
I'm okay with that, removing the final is way easier than adding it later on